nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestEncryptContent.java

```diff
@@ -60,12 +74,97 @@ public void testRoundTrip() throws IOException {
             testRunner.run();
             testRunner.assertAllFlowFilesTransferred(EncryptContent.REL_SUCCESS, 1);
 
+            logger.info("Successfully decrypted {}", method.name());
+
             flowFile = testRunner.getFlowFilesForRelationship(EncryptContent.REL_SUCCESS).get(0);
             flowFile.assertContentEquals(new File("src/test/resources/hello.txt"));
         }
     }
 
     @Test
+    public void testShouldDecryptOpenSSLRawSalted() throws IOException {
```

Comment:
This test failed in build:
[pool-2-thread-1] ERROR org.apache.nifi.processors.standard.EncryptContent - EncryptContent[id=3720f844-b303-4315-9b69-2007297eb6b8] Cannot decrypt FlowFile[0,salted_raw.enc,48B] - : org.apache.nifi.processor.exception.ProcessException: java.security.InvalidKeyException: Illegal key size